### PR TITLE
Extract installer strings for localization

### PR DESCRIPTION
**Pull request by Hans-Halverson**
This PR extracts all text in the DUA for localization. For ease of translation all strings are contained in a single `Localizable.strings` file.

In order to consolidate all strings into a single strings file for easy translations I hooked up a handful of interface components in the interface builder to the AppDelegate using `IBOutlet`. This allows us to set the localized text from code, instead of having a separate system of localized interface builder strings. I've also kept all the comments in the strings file to avoid duplicating them, hence the `nil` argument to all calls to `NSLocalizedString`.

In addition the DUA has a menu with a lot of unnecessary items such as entire File, Edit, View, Window, and Help menus. These appear to be added by default and aren't actually being used by the DUA at all. Users shouldn't need to interact with the menu at all for the DUA, so to avoid having to unnecessarily localize menus I removed all the menus except for the app menu with a single Quit item. This is the only menu item that appears to be useful here, and keeps the app menu nonempty.

This PR also includes the strings for Japanese localization of the installer.

Tests:
Set the app language to the "Accented Pseudolanguage" to test that all strings are being localized.

- Menu now only contains the app menu with a single quit item, which is localized
Screen Shot 2022-05-25 at 3 02 42 PM

- Installer dialog is localized in both download and installing states
Screen Shot 2022-05-25 at 3 02 53 PM
Screen Shot 2022-05-25 at 3 05 04 PM

- All possible error dialogs are localized. (Ignore the fact that some format strings aren't working because the psuedolocale changes the format specifiers).
Screen Shot 2022-05-25 at 3 28 53 PM
Screen Shot 2022-05-25 at 3 02 22 PM
Screen Shot 2022-05-25 at 3 42 43 PM
Screen Shot 2022-05-25 at 3 35 36 PM
Screen Shot 2022-05-25 at 3 36 38 PM

- Set the app language to Japanese and verify that Japanese strings are displayed
Screen Shot 2022-06-08 at 10 34 31 AM

### CODE REVIEW

**Review by mukulhase — APPROVED** [review]
Looks good to me. We can use this commit ID till I get my PR in, so don't merge just yet. I'll unblock you soon.

**Comment by Hans-Halverson** [comment]
Sounds good. I'm waiting until we have the first round of translations to actually merge this so its not pressing.

**Comment by Hans-Halverson** [comment]
Japanese strings have been translated and are now included in this PR. This PR is now ready to go.

**Pull request merged**
